rollup-bridge-contracts/contracts/bridge/libraries/Queue.sol

```diff
@@ -0,0 +1,92 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.27;
```

Comment:
Let's split this huge PR into:

1. NilRollUp only
2. Bridge implementation (on top of first one)